<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>711</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="plotWidget_live" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="plot_data_live"/>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>320</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>711</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AllEditTriggers</set>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>150</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Parameter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
     <item>
      <property name="text">
       <string>New Subitem</string>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="background">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>3</red>
         <green>12</green>
         <blue>3</blue>
        </color>
       </brush>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QListView" name="list_timetraces">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>350</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="txt_tag">
       <property name="text">
        <string>some tag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clear_record_fpga">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_to_disk">
       <property name="text">
        <string>Save to disk</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="plotWidget_sweep" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>401</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="plot_data_sweep"/>
   </widget>
   <widget class="QPushButton" name="btn_stop_record">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>320</y>
      <width>107</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <zorder>listView</zorder>
   <zorder>plotWidget_live</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>progressBar</zorder>
   <zorder>btn_start</zorder>
   <zorder>treeWidget</zorder>
   <zorder>list_timetraces</zorder>
   <zorder>plotWidget_sweep</zorder>
   <zorder>btn_stop_record</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
